### DISCUSSION TITLE
lscpu: last cpu(cluster) is not displayed

### DISCUSSION THREAD

**Discussion by MatejMagat305** [termux/termux-packages#21329] [Q&A]
I have same problem as https://github.com/util-linux/util-linux/issues/3062, but on termux "native", should I open issue? I have snapdragon 8 gen 1 on sm-x706b and it print(not print Cortex-X2):
```
$ lscpu --extended=MODELNAME
MODELNAME
Cortex-A510
Cortex-A510
Cortex-A510
Cortex-A510
Cortex-A710
Cortex-A710
Cortex-A710
-
```

**Comment by sylirre** [comment]
Looks like it fails to recognize certain CPU part values in /proc/cpuinfo. This can be an issue of `lscpu` implementation as well.
